programs/drift-competitions/src/instructions/receive_randomness.rs

```diff
@@ -0,0 +1,35 @@
+use anchor_lang::prelude::*;
+use switchboard_solana::prelude::*;
+use crate::state::Competition;
+
+pub fn receive_randomness(_ctx: Context<ReceiveRandomness>, winner_randomness: u128, prize_randomness: u128) -> Result<()> {
+    msg!("winner_randomness {}", winner_randomness);
+    msg!("prize_randomness {}", prize_randomness);
+
+    let mut competition = _ctx.accounts.competition.load_mut()?;
+    competition.winner_randomness = winner_randomness;
+    competition.prize_randomness = prize_randomness;
```

Comment:
i named these "draw" in the competition structure, should we just set those here?